mbank_parser.py

```diff
@@ -0,0 +1,155 @@
+import csv
+import constants
+
+YNAB_HEADERS = constants.YNAB_HEADERS
+YNAB_FILENAME_PREFIX = constants.YNAB_FILENAME_PREFIX
+YNAB_DELIMITER = constants.YNAB_DELIMITER
+MBANK_DELIMITER = constants.MBANK_DELIMITER
+
+SPLATA_KART = constants.SPLATA_KART
+KARTA = constants.KARTA
+KAPITALIZACJA = constants.KAPITALIZACJA
+PODATEK_ODSETKI = constants.PODATEK_ODSETKI
+INTERNAL_TRANSFER = constants.INTERNAL_TRANSFER
+INTERNAL_INCOMING = constants.INTERNAL_INCOMING
+REPAYMENT = constants.REPAYMENT
+OPERATION_HEADER = constants.OPERATION_HEADER
+OWNER = constants.OWNER
+
+KONTO_1 = constants.KONTO_1
+KONTO_2 = constants.KONTO_2
+KONTO_3 = constants.KONTO_3
+KONTO_REGULARNE = constants.KONTO_REGULARNE
```

Comment:
Too much repetition. Maybe just `from constants import *` if you really want to get rid of `constats.`. or `import constants as c` and use those as `c.SPLATA_KART`?